mods/lord/_experimental/lord_bows/src/bows/api.lua

```diff
@@ -0,0 +1,105 @@
+require("bows.behavior.mechanics_throwing")
+local entity_projectiles = require("bows.behavior.entity_projectiles")
+
+local bows = {
+	--- @type table<bows.BowRegistration>
```

Comment:
не указал тип ключа:
```suggestion
	--- @type table<string,bows.BowRegistration>
```
https://emmylua.github.io/annotations/dict.html

Reply:
в остальных местах тоже глянь